### ISSUE TITLE
Add extension-storage as a well-known collection (or whatever is needed here, if anything)

### ISSUE THREAD

**Issue by thomcc** [mozilla-services/syncstorage-rs#367]
This is planned sometime during Q1 probably, and is currently in progress (based on generic sync). It should be set up to use whatever (if any) optimizations are performed for other collections at some point prior to it going live. There's no rush here, but it's mostly a heads up.

This is intended to be roughly equivalent to this legacy server bug: https://github.com/mozilla-services/server-syncstorage/issues/128

**Comment by pjenvey** [comment]
We'll likely be able to safely add this to syncstorage-rs's std collections, but note that [it doesn't seem easy to add to python server-syncstorage](https://github.com/mozilla-services/server-syncstorage/pull/133#issuecomment-571815716).

**Comment by tublitzed** [comment]
We don't _think_ need this anymore, but will wait on @Micheletto or @rfk to weigh in before closing as a won't fix.

**Comment by rfk** [comment]
Per meeting: WONTFIX
